inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -132,10 +132,19 @@ class CNNLayerCreator : public ::ngraph::AttributeVisitor {
 
     void on_adapter(const std::string& name, ::ngraph::ValueAccessor<void>& adapter) override;
 
+    void on_adapter(const std::string& name, ::ngraph::ValueAccessor<void*>& adapter) override {
+        if (std::string(node->get_type_name()) != "Constant") {
+            const auto data_beg = static_cast<char*>(adapter.get_ptr());
+            const auto data_end = std::next(data_beg, adapter.size());
+            params[name] = std::string{data_beg, data_end};
```

Comment:
nitpick: Looks like you can simple use: `params[name] = std::string(data_beg, adapter.size());` instead of these 2 lines.

Reply:
Corrected.